### ISSUE TITLE
NVDA version does not announce content from the note-editor

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#3933]
https://forums.zotero.org/discussion/113455/zotero-7-beta-accessibility-of-note-editor

I could not reproduce it at first with my NVDA setup - things were being read out line by line as expected.

But after updating NVDA to the latest version, NVDA stopped reading out the lines as you navigate through the note with arrows.
